.circleci/config.yml

```diff
@@ -245,18 +233,13 @@ jobs:
       - run:
           name: Resolve typescript version
           command: |
-            pnpm add typescript@next -d -w
+            pnpm install typescript@next -d -w
```

Comment:
I think the best command I know for this is

```
pnpm update -r typescript@next
```

it should update every workspace in the repo where typescript is installed, including the root.